workalendar/america/argentina.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from datetime import timedelta, date
+from workalendar.core import WesternCalendar, ChristianMixin
+from ..core import MON, TUE, WED, THU, FRI, SAT, SUN
+from ..registry_tools import iso_register
+
+@iso_register('AR')
+class Argentina(WesternCalendar, ChristianMixin):
+    'Argentina'
+
+    include_good_friday = True
+    include_easter_saturday = True
+    include_easter_sunday = True
+    include_christmas = True
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (3, 24, "Día de la Memoria"),                                       # -- ok
```

Comment:
I guess that those "ok" marks were acting like temporary "checkboxes" on your todo list?... They could all be removed (and it would help your branch pass the flake8 tests, too).